### PR TITLE
Create column-reference for PostgreSQL.

### LINKED ISSUE #4536: [Concept Entry] Postgresql column-references
### Reason/inspiration (optional)

We would like a new entry on the `Column References` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/column-references/column-references.md`.

The entry should include:

- An introductory paragraph that describes the concept
- A ‘Syntax’ section that includes the syntax for the concept (if applicable)
- An ‘Example’ section that shows the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ogumbaiyke**
### Description

I made a summary of column reference and how to use it to get information from databases. Motivation includes just learning and growing in tech.

### Issue Solved

Fixes #4536

Issues solved includes understanding of column reference in simple terms, clear syntax lines of code with good comments that can aid better understanding of topic and usage of syntax and applying column reference.

### Type of Change
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/postgresql/concepts/column-references/column-references.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+           CREATE TABLE table_name (
+             column1 data_type,
+             column2 data_type,
+    
+            );
+
+  
+## Example
+
+This is an example showing how to use column references in PostgreSQL. A table is initially created and column names are given together with data types to input the right data. It also shows how to update data through column references.
```
Comment (Sriparno08) [inline content/postgresql/concepts/column-references/column-references.md:96]:
```suggestion
## Example

The following example demonstrates the usage of column references in PostgreSQL:
```

File: content/postgresql/concepts/column-references/column-references.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Sriparno08) [inline content/postgresql/concepts/column-references/column-references.md:179]:
````suggestion
```sql
-- Creating a table

CREATE TABLE bus (
  first_name VARCHAR(80),
  last_name VARCHAR(80),
  occupation VARCHAR(80),
  starting_point VARCHAR(80),
  destination_point VARCHAR(80),
  ticket_price INTEGER,
  duration_in_minutes BIGINT,
  type_of_payment VARCHAR(90),
  age INTEGER,
  next_of_kin VARCHAR(100),
  date_of_payment DATE  
);

-- Inserting data

INSERT INTO bus (first_name, last_name, occupation, starting_point, destination_point, ticket_price, duration_in_minutes, type_of_payment, age, next_of_kin, date_of_payment)

VALUES 
  ('Ikechukwu', 'Ogumba', 'Student', 'Dei-Dei', 'Kubwa', 200, 15, 'Card Payment', 22, 'William Ogumba', '2024-04-20'),
  ('John', 'Snow', 'Plumber', 'Dei-Dei', 'Wuse Junction', 500, 40, 'Card Payment', 30, 'Micheal Bolton', '2024-04-20'),
  ('Barry', 'Hickler', 'Banker', 'Dei-Dei', 'Berger', 700, 60, 'Cash Payment', 39, 'James Rashford', '2024-04-20'),
  ('Jenny', 'Simpson', 'Therapist', 'Dei-Dei', 'Maitama', 800, 80, 'Card Payment', 30, 'Julie Simpson', '2024-04-20'),
  ('Junior', 'Kelechukwu', 'Student', 'Dei-Dei', 'Kubwa', 200, 15, 'Card Payment', 40, 'Matthew Kelechukwu', '2024-04-20'),
  ('Folakemi', 'Abimbola', 'Pastor', 'Dei-Dei', 'Wuse Junction', 500, 40, 'Cash', 30, 'Kehinde Abimbola', '2024-04-20'),
  ('Tochukwu', 'Okafor', 'Athlete', 'Dei-Dei', 'Berger', 700, 60, 'Card Payment', 39, 'Duru Okafor', '2024-04-20'),
  ('Taiwo', 'Kehinde', 'Trader', 'Dei-Dei', 'Kubwa', 200, 15, 'Card Payment', 50, 'Adebayo Kehinde', '2024-04-20');

SELECT
  first_name,
  last_name,
  age

FROM
  bus;
```

The above code lists the first name, last name and age of all the passengers in the bus.
````

**Review by Sriparno08 — COMMENTED** [review]
Hey @ogumbaiyke, thanks for contributing to Codecademy!

I've listed all the modifications that you need to make in your entry below:

**Comment by ogumbaiyke** [comment]
Thanks. I have made the required changes.

**Review by mohitO12-dev — APPROVED** [review]
approved

**Review by Sriparno08 — APPROVED** [review]
Looks good for a second review! 👍

**Review by mohitO12-dev — APPROVED** [review]
approved

**Review by mohitO12-dev — APPROVED** [review]

**Review by mohitO12-dev — APPROVED** [review]

**Pull request merged**